installation-pc.md

```diff
@@ -0,0 +1,134 @@
+CALOUR installation instructions for windows 10
+===============================================
+
+Install miniconda (if already installed - skip this step):
+----------------------------------------------------------
+Go to [https://conda.io/miniconda.html](https://conda.io/miniconda.html), download the Python 3.6 64 bit exe installer, and run it.
+
+You can select all the default options in the installer.
+
+Create the Calour conda environment
+-----------------------------------
+In the windows start menu, select "anaconda prompt". You will get a command prompt.
+
+Create a [conda](http://conda.pydata.org/docs/install/quick.html) environment for calour:
+```
+conda create -n calour python=3.5 matplotlib numpy scipy pandas qt jupyter scikit-learn statsmodels
+```
+(answer 'y' to the installation question)
+
+and activate it using:
+```
+activate calour
+```
+
+Install biom-format
+-------------------
+Try to run the command:
+```
+pip install biom-format
+```
+If it fails (Error and then red colored text), it means you need to install the Microsoft Build Tools 2015 as follows:
+
+go to [http://landinghub.visualstudio.com/visual-cpp-build-tools](http://landinghub.visualstudio.com/visual-cpp-build-tools), download "Build tools 2015", and run the installer (You can select all default options).
+
+Then retry:
+```
+pip install biom-format
+```
+
+Install scikit-bio dependency:
+```
+conda install -c biocore biom-format scikit-bio
```

Comment:
why `biom-format` again?

Reply:
oops. nice catch :)
fixed to pip install since this is windows